pipelines/batch/src/main/java/org/openmrs/analytics/ConvertResourceFn.java

```diff
@@ -76,6 +79,24 @@ public void writeResource(HapiRowDescriptor element)
             .setVersionId(element.resourceVersion())
             .setLastUpdated(simpleDateFormat.parse(element.lastUpdated()));
 
+    if (element.getTags() != null) {
+      List<Coding> tags = new ArrayList<>();
+      List<CanonicalType> profiles = new ArrayList<>();
+      for (ResourceTag resourceTag : element.getTags()) {
+        // The HAPI FHIR tagType of value 0 means it's of type TAG, 1 for PROFILE and 2 for SYSTEM.
+        // https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-jpaserver-model/ca/uhn/fhir/jpa/model/entity/TagTypeEnum.html
```

Comment:
Any reasons why we are ignoring tagType of value 2, they seem to be of tagType SECURITY_LABEL.

Reply:
Yeah, I did not find anything related to this in the database. Given if there was something in the database, we could have figured where its supposed to be in FHIR response or parquet file.